frontend/src/components/GoalReward.tsx

```
@@ -30,8 +41,8 @@ export default function GoalReward({ goal, onContinue }: GoalProps) {
                     <p>{goal.title}</p>
                     <p>You earned 5 gems</p>
                 </div>
-                <button onClick={handleContinue} className={styles.text}>
-                    Keep it Going!
+                <button onClick={handleContinue} className={styles.text} disabled={isProcessing}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)